.github/workflows/release-pr.yml

```diff
@@ -7,7 +7,7 @@ env:
   DOCKER_USERNAME: "grafana"
   DRY_RUN: false
   IMAGE_PREFIX: "trevorwhitney075"
-  RELEASE_LIB_REF: "release-1.14.x"
+  RELEASE_LIB_REF: "release-1.17.x"
```

Comment:
I wonder if we want this as `main` for merging into `main`. That way we don't fall so far behind again, thoughts?